### DISCUSSION TITLE
`watchgod` in `requirements.txt`

### DISCUSSION THREAD

**Discussion by mtelka** [Kludex/uvicorn#2441] [Q&A]
There is `watchgod` listed in the `requirements.txt` file.  Since `watchgod` got renamed to `watchfiles` more than two years ago I think it should be changed in the `requirements.txt` file too.

Is there any reason to keep `watchgod` in the `requirements.txt` file?

**Comment by Kludex** [comment]
We still maintain the `watchgod` reloader.
